OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/KeyserverSyncAdapterService.java

```diff
@@ -67,13 +70,14 @@
 
 
     private static final String ACTION_IGNORE_TOR = "ignore_tor";
-    private static final String ACTION_UPDATE_ALL = "update_all";
+    public static final String ACTION_UPDATE_ALL = "update_all";
     private static final String ACTION_SYNC_NOW = "sync_now";
     private static final String ACTION_DISMISS_NOTIFICATION = "cancel_sync";
     private static final String ACTION_START_ORBOT = "start_orbot";
     private static final String ACTION_CANCEL = "cancel";
 
     private AtomicBoolean mCancelled = new AtomicBoolean(false);
+    Context context;
```

Comment:
Unused variable? Please make sure you correct/explain all the warnings shown by Studio.